docs/INSTALL.md

```diff
@@ -26,6 +26,7 @@ You need the following dependencies.
 * Enable virtualization in the BIOS
 * Hugepages mounted at `/dev/hugepages` or updated location in [`scripts/docker_setup.sh`](../scripts/docker_setup.sh)
   - Reserve 1G HugePage and disable Transparent HugePages by setting: `default_hugepagesz=2G hugepagesz=2G hugepages=<# of hugepages> transparent_hugepage=never`, where <# of hugepages> = 2 x number of BESS UPF instances
+  - ```sudo sysctl -w vm.nr_hugepages=102400```
```

Comment:
Actually, this depends on the type of system. Please see [here](https://github.com/omec-project/bess/wiki/Build-and-Install-BESS#start-hugepages)
```
# For single-node systems
sudo sysctl vm.nr_hugepages=1024

# For multi-node (NUMA) systems
echo 1024 | sudo tee /sys/devices/system/node/node0/hugepages/hugepages-2048kB/nr_hugepages
echo 1024 | sudo tee /sys/devices/system/node/node1/hugepages/hugepages-2048kB/nr_hugepages
```

BTW, why such as large number of hugepages (102400)?